### ISSUE TITLE
'App called -statusBar or -statusBarWindow on UIApplication'

### ISSUE THREAD

**Issue by mrshawnj** [jasonelle-archive/jasonelle-v2#49]
I downloaded the iOS and tried to run the demo in xcode and got this:

*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'App called -statusBar or -statusBarWindow on UIApplication: this code must be changed as there's no longer a status bar or status bar window. Use the statusBarManager object on the window scene instead'

Does this happen for anyone else?

**Comment by clsource** [comment]
Please provide more details.

Which Xcode Version are you using?
Which iOS version are you targeting?
Which Json are you testing?
Are you using the latest code from the branch develop in jasonelle/jasonette-ios ?

Thanks

**Comment by clsource** [comment]
This seems a new issue regarding iOS 13 

https://medium.com/@hacknicity/view-controller-presentation-changes-in-ios-13-ac8c901ebc4e

Try using Xcode 10 to build the apps until a fix is found.

**Comment by mrshawnj** [comment]
xcode 10 works, thank you

**Comment by mrshawnj** [comment]
Revisiting this problem bc now seems Xcode 10 isn't available to download anymore. Same issue above in this:

//  Copyright © 2016 gliechtenstein. All rights reserved.
//
#import <UIKit/UIKit.h>
#import "JasonAppDelegate.h"

int main(int argc, char * argv[]) {
    @autoreleasepool {
        return UIApplicationMain(argc, argv, nil, NSStringFromClass([JasonAppDelegate class]));
    }
}

**Comment by clsource** [comment]
Is this using the latest version in the repo https://github.com/jasonelle/jasonette-ios ?

**Comment by mrshawnj** [comment]
Yes I think it was, but let me double check everything, I will try latest
and latest xcode 13 and get back to you
